### DISCUSSION TITLE
Woensdag 04-12-24

### DISCUSSION THREAD

**Discussion by Nazneen05x** [rutgerkock/buurtcampus-oost#15] [Stand-ups]
- Hoe gaat het met je en waar ben je nú mee bezig?
- Wat wil je deze week bereiken (aan werken)?
- Waar loop je tegen aan?

**Comment by Nazneen05x** [comment]
**Hoe gaat het met je en waar ben je nú mee bezig?**

I am okay, last van verkoudheid. Ik ben deze week begonnen met de over pagina. De pagina is geïmplementeerd en gestyled. 

**Wat wil je deze week bereiken (aan werken)?**

Ik had maandag aangeven dat ik de over pagina wil beginnen en afmaken deze week. Ik ben goed op weg. De mobiele versie is nu nog alleen op desktop en tablet, dan ben ik klaar met het bouwen gedeelte, daarna nog alleen maar testen en dan eventueel verbeteren.  

**Waar loop je tegen aan?**

Ik heb gisteren een mail gehad van de opdrachtgever met de nieuwe pagina indeling. De Over content staat nu in de Contact pagina. Ik heb de opdrachtgever een bericht gestuurd met mijn mening en vraag. Ik ga alsnog deze pagina afronden, kan het nog altijd veranderen.

**Comment by Annevd** [comment]
**Hoe gaat het met je en waar ben je nú mee bezig?**

Het gaat wel oké. Ik ben nu nog steeds bezig met de deeltaak creative coding. Ook heb ik een aantal issues vanuit het projectplan aangemaakt en in het projectboard gezet.

**Wat wil je deze week bereiken (aan werken)?**

Sowieso wil ik eerst de creative coding opdracht af hebben, en daarna weer verder aan de must haves/should haves van de contact pagina (omdat die nu hogere prioriteit heeft dan de zaden pagina)

**Waar loop je tegen aan?**

Ik ben een beetje gestrest over het combineren van de deeltaak en het projectplan. Ik heb er heel weinig tijd voor. Vooral de zaden pagina is nog verre van af.

**Comment by wingsvn** [comment]
**hoe gaat het met je en waar ben je nu mee bezig?**
Het gaat niet heel lekker. Ik ben een beetje overweldigt en weet niet meer waar ik moet beginnen.

**Wat wil de deze week bereiken (aan werken)?**
werken aan de creative coding deeltaak.
 
**Waar loop je tegen aan?**
Ik voel me ook gestresst met alles combineren.

**Comment by Lmikkers** [comment]
### Hoe gaat het met je en waar ben je nú mee bezig?
Het gaat oke ik ben bezig met de deeltaak opdracht, merk alleen wel dat ik minder motivatie heb met school. Misschien doordat alles door elkaar gaat en ik geen goed overzicht heb.  
Ook moet ik vrijdag mijn herkansing inleveren, dus ben hier druk mee bezig en heeft dit mijn prioriteit. Ook moet ik maandag mijn stage opdracht afhebben, dus eerst focus op herkansing, dan stageopdracht en dan weer schoolopdracht.

### Wat wil je deze week bereiken (aan werken)?
Aan mijn deeltaak 

### Waar loop je tegen aan?
Combineren van bieb in bloei met deeltaak en de bieb in bloei planning. Herkansing moet ik goed inleveren
